src/main/java/com/ibm/disni/verbs/RdmaConnParam.java

```diff
@@ -221,4 +225,23 @@ public int getQp_num() {
 	public void setQp_num(int qp_num) throws IOException {
 		throw new IOException("Operation currently not supported");
 	}
+
+
+	public void writeBack(ByteBuffer buffer) {
+		buffer.order(ByteOrder.LITTLE_ENDIAN);
+		buffer.putLong(private_data_addr);
```

Comment:
private_data_addr should be in native byte order

Reply:
Thanks. The code was fine for me as I mostly work with little endian cpus.